setup.py

```diff
@@ -21,7 +21,7 @@
     CONFIG_VERSION = f'\\"{CONFIG_VERSION}\\"'
     # Make sure that pthreads is linked statically, otherwise we run into problems
     # on computers where it is not installed.
-    extra_link_args = ["-Wl,-Bstatic", "-lpthread"]
+    extra_link_args = ["-Wl,-Bstatic", "-lpthread", "-static"]
```

Comment:
This `-static` is applied globally, arguments like below are more flexible:
```
extra_link_args = ["-Wl,-Bstatic", "-lgcc", "-lpthread"]
```

Reply:
What would be the difference in this case? The resulting binary looks fine.

Reply:
The result has no difference in this case. But, if using `-static`, other arguments are not necessary. Like this:
```
extra_link_args = ["-static"]
```
Current code makes people feel confused, I think so.

Doc: https://gcc.gnu.org/onlinedocs/gcc/Link-Options.html

Reply:
Thanks! It works.

Reply:
Thanks you too! Your excellent works and makes me keen on study. :smile: